### PR TITLE
XW-4876 | support for recommended playlist

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-4876
@Wikia/x-wing

### CODE REVIEW

File: app/components/recommended-video.js [hunk 1]
```diff
@@ -74,6 +74,8 @@ export default Component.extend(NoScrollMixin, {
 			jwPlayerAssets.load()
 		]).then(([videoData]) => {
 			if (!this.get('isDestroyed')) {
+				const shuffledPlaylist = videoData.playlist.sort(() => 0.5 - Math.random());
```
Comment (bkoval) [inline app/components/recommended-video.js:77]: New line after const :)

Comment (bkoval) [reply app/components/recommended-video.js:77]: Nice randomizer btw :) 👍

**Review by bkoval — APPROVED** [review]

File: app/components/recommended-video.js [hunk 2]
```diff
@@ -145,7 +147,7 @@ export default Component.extend(NoScrollMixin, {
 	},
 
 	getVideoData() {
-		return fetch(`https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}`).then((response) => response.json());
+		return fetch(`https://cdn.jwplayer.com/v2/playlists/${this.get('playlistId')}?related_media_id=${this.get('relatedMediaId')}`).then((response) => response.json());
```
Comment (bkoval) [inline app/components/recommended-video.js]: This line is too long for linter :)

**Review by rybmat — APPROVED** [review]

File: app/components/recommended-video.js (hunk 1 above)
Comment (latata) [reply app/components/recommended-video.js:77]: stackoverflow btw ;)

**Pull request merged**